gui/src/containers/JobSpecRuns.js

```diff
@@ -23,47 +25,54 @@ const styles = theme => ({
   title: {
     marginTop: theme.spacing.unit * 5,
     marginBottom: theme.spacing.unit * 5
+  },
+  customButtons: {
+    flexShrink: 0,
+    color: theme.palette.text.secondary,
+    marginLeft: theme.spacing.unit * 2.5,
   }
 })
 
-const START_PAGE = 1
-
 export class JobSpecRuns extends Component {
-  constructor (props) {
+  constructor(props) {
     super(props)
     this.state = {
       page: 0
     }
     this.handleChangePage = this.handleChangePage.bind(this)
   }
 
-  componentDidMount () {
-    const {jobSpecId, pageSize, fetchJobSpecRuns} = this.props
-    fetchJobSpecRuns(jobSpecId, START_PAGE, pageSize)
+  componentDidMount() {
+    const { jobSpecId, pageSize, fetchJobSpecRuns } = this.props
+    if (this.props.match.params.jobRunsPage) {
+      const START_PAGE = this.props.match.params.jobRunsPage
+      this.setState({ page: START_PAGE - 1 })
+      fetchJobSpecRuns(jobSpecId, START_PAGE, pageSize)
+    } else {
+      this.setState({ page: 0})
+      fetchJobSpecRuns(jobSpecId, 1, pageSize)
+    }
   }
 
-  handleChangePage (e, page) {
-    const {fetchJobSpecRuns, jobSpecId, pageSize} = this.props
-
+  handleChangePage(e, page) {
+    const { fetchJobSpecRuns, jobSpecId, pageSize } = this.props
+    this.props.history.replace(`/job_specs/${jobSpecId}/runs/page/${page+1}`)
     fetchJobSpecRuns(jobSpecId, page + 1, pageSize)
-    this.setState({page})
+    this.setState({ page })
   }
-
-  render () {
-    const {classes, jobSpecId} = this.props
+  render() {
+    const { classes, jobSpecId } = this.props
 
     return (
       <div>
         <Breadcrumb className={classes.breadcrumb}>
-          <BreadcrumbItem href='/'>Dashboard</BreadcrumbItem>
+          <BreadcrumbItem href="/">Dashboard</BreadcrumbItem>
```

Comment:
Our linter enforces single quotation marks for js strings. Any particular reason for the switch to the double quotation marks?

Reply:
Forgot to lint, my bad 😊